packages/semantic-release-config/index.js

```diff
@@ -1,43 +1,55 @@
 module.exports = {
   plugins: [
     [
-      "@semantic-release/commit-analyzer",
+      '@semantic-release/commit-analyzer',
       {
-        preset: "conventionalcommits",
+        preset: 'conventionalcommits',
         releaseRules: [
-          { type: "docs", release: "patch" },
-          { type: "refactor", release: "patch" },
-          { type: "style", release: "patch" },
-          { type: "test", release: "patch" },
-          { type: "ci", release: "patch" },
-          { type: "build", release: "patch" },
+          { type: 'docs', release: 'patch' },
+          { type: 'refactor', release: 'patch' },
+          { type: 'style', release: 'patch' },
+          { type: 'test', release: 'patch' },
+          { type: 'ci', release: 'patch' },
+          { type: 'build', release: 'patch' },
```

Comment:
Same with `build`. I don't think we can release on `build` and then not have it be in the `release-notes-generator` as couldn't this lead to a release due to a `build` change but it not being included in the release notes?

Reply:
Should we also exclude `ci` from release and notes?